pangraph/Tuura/Pangraph/Options.hs

```diff
@@ -0,0 +1,75 @@
+module Tuura.Pangraph.Options (Options(..), getOptions) where
```

Comment:
This also seems to be only related to the executable, so let's put it into `Fantasi`.